docs/design-docs/rook-ceph/README.md

```diff
@@ -0,0 +1,211 @@
+## Replication / configuration
+
+Configuration data is stored in location: /var/lib/ceph
+Storage data is located on dedicated devices which are connected via OSD pods.
+
+Replication: Like Ceph Clients, Ceph OSD Daemons use the CRUSH algorithm, but the Ceph OSD Daemon uses it to compute where replicas of objects should be stored (and for rebalancing). In a typical write scenario, a client uses the CRUSH algorithm to compute where to store an object, maps the object to a pool and placement group, then looks at the CRUSH map to identify the primary OSD for the placement group.
+The client writes the object to the identified placement group in the primary OSD. Then, the primary OSD with its own copy of the CRUSH map identifies the secondary and tertiary OSDs for replication purposes, and replicates the object to the appropriate placement groups in the secondary and tertiary OSDs (as many OSDs as additional replicas), and responds to the client once it has confirmed the object was stored successfully.
+
+## Prerequisite
+
+
+Since version 1.4 lvm package present on the nodes is required.
+Example installation command:
+
+RHEL:
+
+`yum install lvm2 -y`
```

Comment:
isn't it done by epiphany already?

Reply:
good question, if no, should we add here also command for ubuntu? sudo apt-get install lvm2?

Reply:
Let's not bother about it.